lib/rules/selector-max-id/__tests__/index.mjs

```diff
@@ -236,11 +236,11 @@ testRule({
 		{
 			code: '#foo { &:hover > #bar #baz {} }',
 			description: 'nested selectors: greater than max ID selectors',
-			message: messages.expected('#foo:hover > #bar #baz', 2),
+			message: messages.expected('&:hover > #bar #baz', 2),
 			line: 1,
 			column: 8,
 			endLine: 1,
-			endColumn: 30,
+			endColumn: 27,
 		},
 		{
 			code: '#foo #bar #baz { @media print { #bar {} } }',
```

Comment:
question

Are you sure this is valid CSS?
i.e. `@media` can't be nested in that case
I suppose it's SASS.

> The main extensions compared to level 2 are allowing nesting of certain at-rules inside [@media](https://www.w3.org/TR/css-conditional-3/#at-ruledef-media)…

Reply:
This is certainly valid CSS :)

Reply:
The CSS nesting spec can be found here : https://drafts.csswg.org/css-nesting/